src/HolographFactory.sol

```diff
@@ -228,6 +228,15 @@ contract HolographFactory is Admin, Initializable, Holographable, HolographFacto
     if (v < 27) {
       v += 27;
     }
+    if (v != 27 && v != 28) {
```

Comment:
Could you add a note about this logic and hashes.

I found some documentation that looks related in the OZ repo, but this is a bit different:

```
        // EIP-2 still allows signature malleability for ecrecover(). Remove this possibility and make the signature
        // unique. Appendix F in the Ethereum Yellow paper (https://ethereum.github.io/yellowpaper/paper.pdf), defines
        // the valid range for s in (301): 0 < s < secp256k1n ÷ 2 + 1, and for v in (302): v ∈ {27, 28}. Most
        // signatures from current libraries generate a unique signature with an s-value in the lower half order.
        //
        // If your library generates malleable signatures, such as s-values in the upper range, calculate a new s-value
        // with 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141 - s1 and flip v from 27 to 28 or
        // vice versa. If your library also generates signatures with 0/1 for v instead 27/28, add 27 to v to accept
        // these malleable signatures as well.
        ```

Reply:
Notes and details added in latest commit. Also updated and documented the functionality in `enforcer/HolographERC20.sol`.

Reply:
Nice!